docs/guides/qiskit-function-templates.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Qiskit Function templates
+description: Learn how to use Qiskit Code Assistant, a generative AI code assistant.
+---
+{/* cspell:ignore FCIDUMP */}
+
+
+# Introduction to Qiskit Function templates
+
+The [Qiskit Function templates](https://github.com/qiskit-community/qiskit-function-templates) provide realistic examples for how to build utility-scale application workflows with Qiskit and the Qiskit addons. These prebuilt workflows are customizable for a variety of research needs and can be used instead of starting from scratch. To simplify execution and parallelization, they are easily deployable to [Qiskit Serverless](./serverless).
+
+The templates provided are broken into two basic categories:
```

Comment:
Maybe it's worth walking through what's included in a Base Template pipeline, so you can further illustrate what's interesting about the Application template?